lib/matplotlib/axis.py

```diff
@@ -164,14 +163,10 @@ def __init__(self, axes, loc, label=None,
             GRIDLINE_INTERPOLATION_STEPS
         self.label1 = mtext.Text(
             np.nan, np.nan,
-            fontproperties=font_manager.FontProperties(size=labelsize),
```

Comment:
there are a few cleanups like this which are technically independent and could be extracted to a separate PR if needed.

Reply:
I always like smaller PRs better :smile:.

Reply:
Sure, it was split into https://github.com/matplotlib/matplotlib/pull/16246.